### PR TITLE
Prepare release 30.2.0

### PR DESCRIPTION
**Pull request by pombredanne**
This PR is to prepare  release 30.2.0 with these changes:

This is a critical bug fix release including features updates.

- We fixed critical performance issues that made the web UI unusable. This include
  removing some less interesting redundant details displayed in the web UI for
  vulnerabilities.
- We made minor documentation updates.
- We re-enabled support for Arch linux, Debian, and Ubuntu security advisories importers
- We added a new improver for Oval data sources
- We improved Alpine linux and Gitlab security advisories importers 

Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -33,13 +33,13 @@ join our community. Below are some examples to get involved:
...
 
-You are here to help, but you're a new contributor! No worries, we always
+You are here to help, but you are a new contributor! No worries, we always
 welcome newcomer contributors. We maintain some
 `good first issues <https://github.com/nexB/vulnerablecode/labels/good%20first%20issue>`_
 and encourage new contributors to work on those issues for a smooth start.
 
 .. tip::
-    If you're an open-source newbie, make sure to check the extra resources at
+    If you arere an open-source newbie, make sure to check the extra resources at
```
Comment (TG1999) [inline docs/source/contributing.rst:42]:
```suggestion
    If you are an open-source newbie, make sure to check the extra resources at
```

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ Other Ways
 ^^^^^^^^^^
 
 You want to contribute to other aspects of the VulnerableCode project, and you
-can't find what you're looking for! You can always discuss new topics, ask
+cannot find what you are looking for! You can always discuss new topics, ask
```
Comment (TG1999) [inline docs/source/contributing.rst:78]:
```suggestion
can not find what you are looking for! You can always discuss new topics, ask
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
-- An overview of VulnerableCode
+VulnerableCode is also a free and open source software (FOSS) project that
+provides the tools to build this open database. The tools handle collecting,
+aggregating and correlating these vulnerabilities and relating them to a correct
+package version. Our project also supports a public cloud instance of this
+database - VulnerableCode.io.
+
+In this documentation you will find information on:
+
+- An overview of VulnerableCode and hwat you can do with it
```
Comment (TG1999) [inline docs/source/index.rst:15]:
```suggestion
- An overview of VulnerableCode and what you can do with it
```

File: docs/source/contributing.rst (hunk 1 above)
Comment (pombredanne) [reply docs/source/contributing.rst]: Committed. Thanks!

**Review by TG1999 — APPROVED** [review]
LGTM! Some nits for your consideration

File: vulnerablecode/__init__.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 import warnings
 from pathlib import Path
 
-__version__ = "30.1.1"
+__version__ = "30.20"
```
Comment (TG1999) [inline vulnerablecode/__init__.py:15]:
```suggestion
__version__ = "30.2.0"
```

**Pull request merged**